pages/Unlock.tsx

```diff
@@ -11,7 +11,14 @@ export function Unlock() {
   const [isUnlocking, setIsUnlocking] = React.useState(false);
   const { signIn } = useSession();
 
-  const handleUnlock = React.useCallback(async (): Promise<void> => {
+  const handleUnlock = useCallback(async () => {
+    // The call `signIn()` below triggers a re-render of the component
+    // and would prompt the user again (before the actual redirect
+    // happens and the view is replaced)
```

Comment:
Ah I was wondering why this happens, I thought it only happens during dev, thanks for the fix 🙌

Reply:
What made you think it only happens during dev? Please properly test these things, I guess this was introduced during the eslint update.

Reply:
It didn't happen when I installed it from the AppStore, I tried to reproduce several times. But it happened during dev.

And I'm not sure if this is after the eslint update as I used to see it even before that...